server/bin/liquibase.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+# Source functions library
+_prefer_jre="true"
+. /usr/share/java-utils/java-functions
+
+# Source system prefs
+if [ -f /etc/java/liquibase.conf ] ; then
+  . /etc/java/liquibase.conf
+fi
+
+# Source user prefs
+if [ -f $HOME/.liquibaserc ] ; then
+  . $HOME/.liquibaserc
+fi
+
+mkdir -p /var/tmp/candlepin
```

Comment:
There are two contexts where we'll to invoke liquibase with slightly different classpaths:
 - `cpdb` when run from a Satellite installation
 - in a development setup via `server/bin/deploy`

My suggestion for handling this (feel free to take another approach), is to default to the Satellite use-case (i.e. exploded war's deps are in `/var/lib/tomcat/webapps/candlepin/WEB-INF/lib`), and then introduce an environment variable, such as `CP_LIQUIBASE_CLASSPATH`  which can override the default path, and then move some of this logic to `server/bin/deploy`, having `server/bin/deploy` do the extraction and setting `CP_LIQUIBASE_CLASSPATH` appropriately.

Reply:
Done. Classpath can be overridden with `CP_LIQUIBASE_CLASSPATH`.